internal/events/statedir_test.go

```
@@ -0,0 +1,67 @@
+	if !strings.Contains(got, filepath.Join("Library", "Application Support", "gren", "events")) {
+		t.Errorf("unexpected path: %s", got)
+	}
+}
+
+func TestNewEventsFile_CreatesDirAndFile(t *testing.T) {
+	t.Setenv("XDG_STATE_HOME", t.TempDir())
+	path, err := NewEventsFile("post-create", "mybranch")
+	if err != nil {
+		t.Fatalf("unexpected error: %v", err)
+	}
+	defer os.Remove(path)
```

golangci-lint (2.11.4)

[error] Error return value of `os.Remove` is not checked

(errcheck)